tutorials/js/data/events.js

```diff
@@ -0,0 +1,60 @@
+window.data = window.data || {};
+window.data.tutorials = window.data.tutorials || [];
+window.data.tutorials.push({
+  name: 'Events',
+  steps: [
+    {
+      name: 'Event Handling',
+      index: 1,
+      description_html: '<p>Events are defined with the standard <a href=""><code>events</code> hash</a> API when using the Backbone or Ampersand adaptor. If a selector is passed in the event key, however, it can only use a <code>js-</code> prefixed class selector. This optimizes performance when delegating events because under the hood, unlike Backbone or Ampersand, Tungsten.js provides its own event delegation system. By default, all events are delegated from the document. Special events can also be handled by an <a href="">event handler plugin</a>.</p><p>All common event types are supported as well as the special cases outlined in the examples in the following sections:</p><ul><li>Directional Swipe Events</li><li>Intent Events</li><li>Document Bindings</li><li>Window Bindings</li><li>Outside Events</li><li>Submit Data</li></ul>'
+    },
+    {
+      name: 'Directional Swipe Events',
+      index: 2,
+      // template_highlights: [{start: {line: 0, ch: 6}, end: {line: 0, ch: 12}}],
+      template: '<p>Test events in the colored area below.</p>\n<div class="js-event-area pure-u-1" style="height:300px; background-color:#71A3AF;"></div>\n<div>captured: {{message}}</div>',
+      description_html: '<p>Exactly what it sounds like:</p><ul><li>swipeup</li><li>swipedown</li><li>swipeleft</li><li>swiperight</li></ul><p>In order to test this on a desktop browser, it might be necessary to put the browser dev tools into device mode to register swipes correctly.</p>',
+      js: "var View = tungsten.backbone.View;\nvar Model = tungsten.backbone.Model;\n\nvar AppModel = Model.extend({\n  message: '---'\n});\n\nvar AppView = View.extend({\n  events: {\n    'swipeup .js-event-area': 'swipeUp',\n    'swipedown .js-event-area': 'swipeDown',\n    'swipeleft .js-event-area': 'swipeLeft',\n    'swiperight .js-event-area': 'swipeRight'\n  },\n  swipeUp: function() {\n    this.setEventMessage('swipeup');\n  },\n  swipeDown: function() {\n    this.setEventMessage('swipedown');\n  },\n  swipeLeft: function() {\n    this.setEventMessage('swipeleft');\n  },\n  swipeRight: function() {\n    this.setEventMessage('swiperight');\n  },\n  setEventMessage: function(msg) {\n    var self = this;\n    this.model.set('message', msg);\n    clearTimeout(this.messageTimeout);\n    this.messageTimeout = setTimeout(function() {\n      self.model.set('message', '---');\n    }, 1200);\n  }\n});\n\nnew AppView({\n  el: '#app',\n  template: compiledTemplates.app_view,\n  model: new AppModel(),\n  dynamicInitialize: true\n});"
```

Comment:
```var AppModel = Model.extend({ message: '---' });``` works, but not in the way you might be thinking.. extend adds new prototype properties to the Model constructor rather than defining data.

You probably want:
```var AppModel = Model.extend({ defaults: {message: '---' }});```
or just
```var appModel = new Model({message: '---'});```

This repeats on steps 3, 5, 6, and 7